StreamChatClient_v3/Database/DatabaseContainer.swift

```diff
@@ -107,6 +107,32 @@ class DatabaseContainer: NSPersistentContainer {
             }
         }
     }
+    
+    /// Removes all data from the local storage.
+    ///
+    /// - Warning: ⚠️ This is a non-recoverable operation. All data will be lost after calling this method.
+    ///
+    /// - Parameters:
+    ///   - force: If sets to `false`, the method fails if there are unsynced data in to local storage, for example
+    /// messages pedning sent. You can use this option to warn a user about potential data loss.
+    ///   - completion: Called when the operation is completed. If the error is present, the operation failed.
+    func flush(force: Bool, completion: ((Error?) -> Void)? = nil) {
```

Comment:
The name `flush` implies something not completed is being forced to complete (ie flushing the output to the console), I suggest `nuke` for this 😂

Reply:
Yes, the naming is not correct there. I'll replace it with `deleteAllData`, which is much better and more descriptive.

Reply:
fixed